Gruntfile.js

```diff
@@ -55,6 +58,7 @@ const { resolve } = path
 const {
   CI = '',
   LIGHTWEIGHT_CLIENT = 'true',
+  GI_VERSION,
```

Comment:
This variable is no longer needed. Can replace all instances with `process.env.GI_VERSION`

Reply:
That's true, but then the `define` option block will become a bit inconsistent:

```js
  define: {
    'process.env.BUILD': "'web'", // Required by Vuelidate.
    'process.env.CI': `'${CI}'`,
    'process.env.GI_VERSION': `'${process.env.GI_VERSION}'`, // <-- Only line accessing `process.env`!
    'process.env.LIGHTWEIGHT_CLIENT': LIGHTWEIGHT_CLIENT,
    'process.env.NODE_ENV': `'${NODE_ENV}'`,
    'process.env.VUEX_STRICT': VUEX_STRICT
  }

Reply:
OK, we'll keep it, resolving.

Reply:
By the way I should likely fix the `LIGHTWEIGHT_CLIENT` expression missing quotes.